recipes.md

```diff
@@ -3,37 +3,46 @@
 Create this `docker-compose.yml` file
 
 ```yaml
-sonarqube:
-  image: sonarqube
-  ports:
-   - "9000:9000"
-  links:
-   - db
-  environment:
-   - SONARQUBE_JDBC_URL=jdbc:postgresql://db:5432/sonar
-  volumes_from:
-   - plugins
+version: "2"
```

Comment:
@andreas-lundgren-storytel I think that will be valuable to specify minimum required version of `docker-compose` which supports this format.